### DISCUSSION TITLE
How to display specific posts on my homepage

### DISCUSSION THREAD

**Discussion by WindowsXp-Beta** [wangchucheng/hugo-eureka#56] [Q&A] [answered]
Can I display specific posts on my homepage?

I guess the setting option `section` in `pages` could help but I don't know how to use it :sob:

**Comment by wangchucheng** [comment]
You can use `section` to specify which section you want to display. For instance, if you want to display pages in `posts`, you can set `section: posts`.

**Reply by WindowsXp-Beta** [reply]
Thanks.I know what `section` means.
However,I still don't know how to display specific articles in my posts on the homepage.

**Comment by wangchucheng** [comment, marked as answer]
As far as I know, you can not choose specific posts to show on the homepage for now.


________________________________
发件人: Xinpeng Wei <notifications@github.com>
发送时间: Tuesday, February 2, 2021 6:56:29 PM
收件人: wangchucheng/hugo-eureka <hugo-eureka@noreply.github.com>
抄送: C. Wang <wangchucheng@outlook.com>; Comment <comment@noreply.github.com>
主题: Re: [wangchucheng/hugo-eureka] How to display specific posts on my homepage (#56)


Thanks.I know what section means.
However,I still don't know how to display specific articles in my posts on the homepage.

―

**Reply by WindowsXp-Beta** [reply]
Thanks for your reply.
One more question,what's the order of the articles in posts based on?
The publishing time?
Can I customize this?

**Comment by wangchucheng** [comment]
Default is publish time. And you can customize it in different ways, such as weight, etc. You can checkout Hugo’s official docs to see how to configure it.


________________________________
发件人: Xinpeng Wei <notifications@github.com>
发送时间: Tuesday, February 2, 2021 9:06:34 PM
收件人: wangchucheng/hugo-eureka <hugo-eureka@noreply.github.com>
抄送: C. Wang <wangchucheng@outlook.com>; Comment <comment@noreply.github.com>
主题: Re: [wangchucheng/hugo-eureka] How to display specific posts on my homepage (#56)


Thanks for your reply.
One more question,what's the order of the articles in posts based on?
The publishing time?
Can I customize this?

―
